gcp/appengine/frontend_handlers.py

```diff
@@ -194,6 +191,17 @@ def osv_get_ecosystems():
   return sorted([bug.ecosystem[0] for bug in query if bug.ecosystem])
 
 
+@cache.cached(timeout=24 * 60 * 60)
```

Comment:
Do we need to set `key_prefix` here? https://flask-caching.readthedocs.io/en/latest/#caching-other-functions

Reply:
Thanks!